tests/e2e/tools/FFI/ContainerFile.tools

```diff
@@ -0,0 +1,20 @@
+FROM quay.io/centos/centos:stream9
+RUN dnf install -y \
+	dnf-plugin-config-manager \
+	epel-release
+RUN dnf config-manager -y --set-enabled crb
+
+RUN dnf install -y \
+        gcc
+
+RUN mkdir -p /root/tests/FFI/src /root/tests/FFI/bin
+
+COPY ./memory/ASIL/20_percent_memory_eat.c /root/tests/FFI/src/
+COPY ./memory/QM/90_percent_memory_eat.c /root/tests/FFI/src/
+COPY ./disk/file-allocate.c /root/tests/FFI/src/
+
+RUN cd /root/tests/FFI/src && gcc -o ../bin/20_percent_memory_eat 20_percent_memory_eat.c
```

Comment:
Can we just add a high level Makefile to compile all of these.

Reply:
Without test, here an example of Makefile:
```
CC = gcc
CFLAGS = -Wall -g

all: 90_percent_memory_eat 20_percent_memory_eat file_allocate

20_percent_memory_eat: 20_percent_memory_eat.c
	$(CC) $(CFLAGS) -o $@ $<

file-allocate: file-allocate.c
	$(CC) $(CFLAGS) -o $@ $<

90_percent_memory_eat: 90_percent_memory_eat.c
	$(CC) $(CFLAGS) -o $@ $<

clean:
	rm -f file-allocate 20_percent_memory_eat 90_percent_memory_eat

.PHONY: all clean
```

Reply:
LGTM

Reply:
Sure will add it

Reply:
@rhatdan Added make file 
@dougsland preder to maintain slim image for now

Reply:
@Yarboa sure, lets make it fly and during the road we can adjust.